code/game/machinery/portable_turret.dm

```diff
@@ -677,10 +729,10 @@
 	if(faction && L.faction == faction)
 		return TURRET_NOT_TARGET
 
-	if(!emagged && issilicon(L) && check_all == 0)	// Don't target silica, unless told to neutralize everything.
+	if(!emagged && issilicon(L) && check_all == FALSE)	// Don't target silica, unless told to neutralize everything.
 		return TURRET_NOT_TARGET
 
-	if(L.stat && !emagged)		//if the perp is dead/dying, no need to bother really
+	if(L.stat == DEAD && !emagged)		//if the perp is dead no need to bother really

```

Comment:
IIRC this specifically checks general stat, so non-emagged turrets that are not on lethal don't stunlock someone.

Reply:
This line of code checks if you're lying down and if the turret is set to lethal. If it's not on lethal, you're not a target if you're laying down.
```
768	if(L.lying)		//if the perp is lying down, it's still a target but a less-important target
769		return lethal ? TURRET_SECONDARY_TARGET : TURRET_NOT_TARGET
```